src/sbt-test/windows/test-bat-template/build.sbt

```diff
@@ -41,15 +41,15 @@ TaskKey[Unit]("check-script") <<= (stagingDirectory in Universal, name, streams)
   val detailScript:File = {
     val d = dir / "bin" / "detail.bat"
     val out = new java.io.PrintWriter( d , "UTF-8")
-    out.print( scala.io.Source.fromFile(script).mkString.replaceAll("@echo off","@echo on & prompt \\$g ") )
+    out.print( scala.io.Source.fromFile(script).mkString.replaceAll("@echo off","echo on & prompt \\$g ") )
     out.close
     d
   }
   def crlf2cr(txt:String) = txt.trim.replaceAll("\\\r\\\n", "\n")
-  def checkOutputEnv(env:Map[String,String], expectedRC: Int, expected:String, args:String*) = {
+  def checkOutput(testName:String, args:String, expected:String, env:Map[String,String]=Map.empty, expectedRC: Int=0) = {
     val pr = new StringBuilder()
     val logger = ProcessLogger((o: String) => pr.append(o+"\n"),(e: String) => pr.append("error < " + e+"\n"))
-    val cmd = Seq("cmd", "/c", script.getAbsolutePath) ++ args
+    val cmd = Seq("cmd", "/c", script.getAbsolutePath+" "+args)
```

Comment:
Well, java doesn't seem to like the command sequence :(
Thanks for figuring this out